web_gui/acct/INSTALL

```diff
@@ -1,2 +0,0 @@
-Requirements:
-httpd, php-mysql, php-xml, php, php-pdo, php-gd
```

Comment:
These are no longer needed? If so, I'll have to update rbh install documentation.